Cross-language - Add examples to reference Nodes in scene tree (by path) (#9459)
godotengine/godot-docs: tutorials/scripting/cross_language_scripting.rst

        }
    }

Getting nodes from the scene tree by path
-----------------------------------------

Getting C# nodes in scene tree from GDScript
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Getting nodes - using C# code - from GDScript is pretty straightforward

Reviewer: ```suggestion
Getting nodes that have a C# script attached to them in GDScript is straightforward:
```

.. code-block:: gdscript

    var MyCSharpNode = get_node("PathTo/MyCSharpNode")

Getting GDScript nodes in scene tree from C#
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> As C# is statically typed, accessing GDScript Nodes requires just an extra step and use

Reviewer: ```suggestion
As C# is statically typed, accessing GDScript nodes requires an extra step and use
```

> the :ref:`GodotObject <class_Object>` type

Reviewer: ```suggestion
the :ref:`GodotObject <class_Object>` type:
```
